src/Tag/types.ts

```diff
@@ -0,0 +1,18 @@
+import { ColorsTypes } from '../shared/theme.types';
+
+export type SizeTypes = 'small' | 'medium' | 'large';
+
+export type VariantTypes = 'filled' | 'outlined';
+
+export type TagProps = {
```

Comment:
I guess we should extend `React.HTMLAttributes<HTMLDivElement>` and allow any html attribute. Then we can use the ...rest props into the wrapper